src/lib/input/input-container.ts

```diff
@@ -142,14 +143,14 @@ export class MdInputDirective implements AfterContentInit {
 
   constructor(private _elementRef: ElementRef,
               private _renderer: Renderer,
-              @Optional() public _ngControl: NgControl) {
+              @Optional() public _ngControl: NgControl) { }
+
+  ngOnInit() {
```

Comment:
So it is simply the case of moving the initialization into the `ngOnInit` hook?
Is that because the FormControl is normally changed by the "user" in their own `ngOnInit` method?